melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/PDLQuery.java

```diff
@@ -6,21 +6,42 @@ private PDLQuery() {}
 
     static final String HENT_PERSON_QUERY = """
             query($ident: ID!) {
-                hentPerson(ident: $ident) {
-                    navn {
-                        fornavn
-                        etternavn
-                    }
-                    foedsel {
-                        foedselsdato
-                    }
-                    statsborgerskap {
-                        land
-                    }
-                    folkeregisterpersonstatus {
-                        status
-                    }
-                }
-            }
+               hentPerson(ident: $ident) {
+                 navn {
+                     fornavn
+                     etternavn
+                     metadata {
+                         master
+                         endringer {
+                             registrert
+                             type
+                         }
+                     }
+                 }
+                 foedsel {
+                     foedselsdato
+                     metadata {
+                         master
+                         endringer {
+                             registrert
+                             type
+                         }
+                     }
+                 }
+                 statsborgerskap {
```

Comment:
Trengs ikke for `statsborgerskap`?

Reply:
Nei, vi tar alt som trengs uavhengig av kilde, ref https://confluence.adeo.no/display/TEESSI/Personopplysninger+-+bruk+av+PDL
Og opphørte opplysninger skal ikke komme med i responsen med mindre vi spesifiserer at vi ønsker historikk.